hummingbot/market/eterbase/eterbase_active_order_tracker.pyx

```diff
@@ -0,0 +1,328 @@
+# distutils: language=c++
+# distutils: sources=hummingbot/core/cpp/OrderBookEntry.cpp
+
+import logging
+import numpy as np
+from decimal import Decimal
+from typing import Dict
+
+from hummingbot.logger import HummingbotLogger
+from hummingbot.core.data_type.order_book_row import OrderBookRow
+
+_cbpaot_logger = None
```

Comment:
`_cbpaot_logger` is the coinbase pro active order tracker logger – yours could be `_eaot_logger`

Reply:
(same goes for all loggers)